includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2935,17 +2936,20 @@ public function schedule_order_tracking( $order_id, $order = null ) {
 	public function create_intent( WC_Order $order, array $payment_methods, string $capture_method = 'automatic', array $metadata = [], string $customer_id = null ) {
```

Comment:
Late here, but I would remove this method altogether. 

It's no longer used anywhere. Originally, it was added #3465 for `WC_REST_Payments_Orders_Controller`. However, we migrated the create_intention part in this controller during the phase 1. 

https://github.com/Automattic/woocommerce-payments/blob/ced4721528c3fb0a856d18b2aad0d480757914fc/includes/admin/class-wc-rest-payments-orders-controller.php#L384